server/lib/utils/connstr-mssql.js

```diff
@@ -0,0 +1,40 @@
+import _ from "lodash";
+
+export function parseConnectionConfig(settings) {
```

Comment:
I would keep those adapter specific utils in the adapter file. Let's try to keep adapters self contained in their own modules...